src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -269,3 +269,15 @@
     (some #(when (= token-symbol (:symbol %))
              %)
           address-tokens)))
+
```

Comment:
mmhh, can this go into some more categorised utils?

we have common/utils/<some-category>

this file is already a bit of a mess of random functionality :)